synapse/synapse.py

```diff
@@ -256,6 +257,9 @@ def replace_table(self, table_name: str, table: pd.DataFrame):
             results = self.syn.tableQuery(f"select * from {synapse_id}")
             self.syn.delete(results)
 
+            # wait for Synapse to catch up
+            time.sleep(5)
```

Comment:
@andrewelamb 1s might be fine to start with; let's see how it behaves and if we run into issues. Ow 5s may be too much if multiple tables are processed.